docs/Changelog-Platform.md

```diff
@@ -4,6 +4,28 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/)
 
+## <a name="2.0.0" />[2.0.0] - 2025-10-07
+
+See full log [of v3.11.0...v4.0.0](https://github.com/microsoft/testfx/compare/v3.11.0...v4.0.0)
+
+### Added
+
+* Breaking changes: see <https://github.com/microsoft/testfx/issues/5762> for information.
```

Comment:
It's a bit of an effort but I would expand the list here instead of only referring to the ticket.

Reply:
I intended to keep the issue link temporarily until https://github.com/dotnet/docs/pull/48673/ is merged and then link the migration guide here (similar to MSTest changelog)